.meta/sinks/gcp_stackdriver_logging.toml

```diff
@@ -122,3 +122,9 @@ For example, Compute Engine VM instances use the labels `projectId`, \
 """
 
 <%= render("_partials/_tls_connector_options.toml", namespace: "sinks.gcp_stackdriver_logging.options", can_enable: false, can_verify: true) %>
+
+<%= render("_partials/_encoding.toml",
+  namespace: "sinks.gcp_stackdriver_logging.options",
+  encodings: ["default"],
+  default: "default"
```

Comment:
`default` should never be an option. If there is only 1 encoding option please represent that here. I do not mind including the `encoding.type` with one enumeration since it is explicit and tells the user we may add more formats in the future.